pkg/rtcp/transport_layer_nack.go

```diff
@@ -0,0 +1,114 @@
+package rtcp
+
+import (
+	"encoding/binary"
+	"fmt"
+	"math"
+)
+
+// The TransportLayerNack packet informs the encoder about the loss of a transport packet
+type TransportLayerNack struct {
+	// SSRC of sender
+	SenderSSRC uint32
+
+	// SSRC of the media source
+	MediaSSRC uint32
+
+	Nacks []struct {
```

Comment:
Same as above: it might make sense to export this struct as a type instead of using an anonymous struct.